### DISCUSSION TITLE
M1 installation of 1.2.10

### DISCUSSION THREAD

**Discussion by bad-drawer** [envkey/envkey#23] [Q&A] [answered]
I'm having an issue installing envkey-source 1.2.10 on a macbook M1.
I get the following when running `curl -s https://raw.githubusercontent.com/envkey/envkey-source/master/install.sh | bash`:
```
Downloading tarball from Github Releases: https://github.com/envkey/envkey-source/releases/download/v1.2.10/envkey-source_1.2.10_darwin_arm64.tar.gz
mv: rename envkey-source to /usr/local/bin/envkey-source: Permission denied
envkey-source is installed in /usr/local/bin
Installation complete. Info:

bash: line 107: envkey-source: command not found
```
Has anyone had this problem?

**Comment by danenania** [comment, marked as answer]
Hey John, does it help if you prefix the install command with sudo?

**Reply by bad-drawer** [reply]
Unfortunately it does not for me. I get the same error.

**Reply by danenania** [reply]
Ok, it seems the problem is that you don’t have write permission to /usr/local/bin, which you’d generally be expected to have on a mac (which is why the install script doesn’t use sudo).

The top answer here has a few options for giving yourself write permissions: https://superuser.com/questions/717663/permission-denied-when-trying-to-cd-usr-local-bin-from-terminal

**Reply by bad-drawer** [reply]
It turns out that the permissions were fine but on a new install of Monterey there was no .zshrc with the default zsh. It all works now. Thanks

**Reply by danenania** [reply]
Gotcha, glad to hear it's working now!
